test/JDBC/input/BABEL-4935.mix

```diff
@@ -0,0 +1,79 @@
+-- tsql
```

Comment:
We should rename the test file to `sys_database_principals.mix`.

Reply:
Added these test to an existing file.